frontend/lib/utils.ts

```
@@ -540,3 +540,74 @@ export const stripHtmlTags = (html: string) => {
+          if (content.styles?.italic) formatted = `*${formatted}*`;
+          if (content.styles?.underline) formatted = `<u>${formatted}</u>`;
+          if (content.styles?.strike) formatted = `~~${formatted}~~`;
+          if (content.styles?.code) formatted = `\`${formatted}\``;
+          
+          return formatted;
+        })
+        .join("") || "";
+
+      switch (block.type) {
+        case "heading":
+          const level = "#".repeat(block.props.level);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)